app/components/primer/auto_complete.rb

```diff
@@ -83,6 +129,8 @@ class AutoComplete < Primer::Component
     # @param system_arguments [Hash] <%= link_to_system_arguments_docs %>
     def initialize(src:, id:, **system_arguments)
       @id = id
+      @aria_label = system_arguments[:"aria-label"] || system_arguments.dig(:aria, :label)
+      system_arguments.delete(:"aria-label") && system_arguments[:aria]&.delete(:label)
```

Comment:
We need to delete `aria-label` so it doesn't get passed into `<auto-complete>` element. This would violate [aria-allowed-attr][0] because aria attributes are only expected on certain elements and roles.

However, this rule is [excluded in Microsoft Accessibility Insights][1] which makes me wonder if we should also exclude it. 

Keeping this rule as I wait [on this][2]

[0]: https://dequeuniversity.com/rules/axe/4.1/aria-allowed-attr
[1]: https://dequeuniversity.com/rules/axe/4.1/aria-allowed-attr
[2]: https://github.com/github/primer/issues/148#issuecomment-845974804